notebooks/image_classification.ipynb

```diff
@@ -647,13 +647,18 @@
     "    ToTensor,\n",
     ")\n",
     "\n",
+    "class ToHalf(torch.nn.Module):\n",
+    "    def forward(self, tensor):\n",
+    "        return tensor.half()\n",
+    "\n",
     "normalize = Normalize(mean=feature_extractor.image_mean, std=feature_extractor.image_std)\n",
     "train_transforms = Compose(\n",
     "        [\n",
     "            RandomResizedCrop(feature_extractor.size),\n",
     "            RandomHorizontalFlip(),\n",
     "            ToTensor(),\n",
     "            normalize,\n",
+    "            ToHalf(),\n",
```

Comment:
I think you don't need to create a class and you can just use a lambda function:

```suggestion
    "            lambda tensor: tensor.half(),\n",
```